### PR TITLE
Feature/add react exports

### LINKED ISSUE #520: Export React Components directly
## Are you reporting a bug or a feature?

 - [ ] Bug
 - [x] Feature

## Expected Behavior

The Patterns Library exports the  React Components directly

## Actual Behavior
You have to import withReact and the components used in this WebComponent

### PR DESCRIPTION
**Pull request by AndyOGo**
Fixes #520 .

Changes proposed in this pull request:

 - export all wrapped components from one file
 - fixes some minor issues

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update
 
 # How Has This Been Tested?
 
 
 
 # Checklist:
 
 - [x] My code follows the style guidelines of this project
 - [x] I have performed a self-review of my own code
 - [x] I have commented my code, particularly in hard-to-understand areas
 - [x] I have made corresponding changes to the documentation
 - [x] My changes generate no new warnings
 - [ ] I have added tests that prove my fix is effective or that my feature works
 - [ ] New and existing unit tests pass locally with my changes
 - [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/js/react-exports.js [hunk 1]
```diff
@@ -0,0 +1,162 @@
+// First import the withReact integration helper function
+import withReact from './with-react'
```
Comment (AndyOGo) [inline src/js/react-exports.js]: Do you agree with single file, named exports and omitting polyfills (`promise` and `custom-elements-v-1`)?

**Comment by AndyOGo** [comment]
@LucaMele 
I agree.
I have only one concern, how do we deal with modules which do not provide custom elements, like `a-device-state`. Guess all are utils and that's it, right?
And what about `<axa-core>`?

**Comment by LucaMele** [comment]
@AndyOGo , yes all utils and therefore they will be created manually. 

axa core we can keep it as it is as i think where was an issue in refactoring it (Nota Bene, still needs to be fixed one day)

no alphabetical ordering dosnt make sense IMHO.

Thanks andy for this

**Comment by AndyOGo** [comment]
@LucaMele 
Do we want to test `create-amo.js` script before merging?

**Comment by LucaMele** [comment]
its ok we can merge as i wanted to do this: `native-exports.js` and then i can test both

**Comment by LucaMele** [comment]
@AndyOGo code looks like it should work perfectly

**Comment by AndyOGo** [comment]
upps my bad, just fixed it...

**Comment by AndyOGo** [comment]
I tested it quickly and fixed some issues, works now.

**Comment by AndyOGo** [comment]
still broken

**Comment by LucaMele** [comment]
@AndyOGo can we merge it? it works now. I tested it here: https://github.com/axa-ch/patterns-library/pull/578/files

**Pull request merged**